wayang-api/wayang-api-sql/src/main/java/org/apache/wayang/api/sql/calcite/rel/WayangTableScan.java

```diff
@@ -78,7 +78,7 @@ public static WayangTableScan create(RelOptCluster cluster, RelOptTable relOptTa
 
     @Override
     public String toString() {
-        return "Wayang TableScan ["+ getQualifiedName() + "]";
+        return "Wayang TableScan [" + getQualifiedName() + "]";
     }
 
```

Comment:
@2pk03 Do we have a unified code style check?

Reply:
not yet, we should ;)